### ISSUE TITLE
Show with year in the title doesn't post-process files without the year

### ISSUE THREAD

**Issue by Zjaaspoer** [pymedusa/Medusa#9916]
**Describe the bug**
When I add a series with a year in the title (e.g. `Foundation (2021)`) but the file that was downloaded and is being post-processed doesn't contain the year name `foundation.s01e02.hdr.2160p.web.h265-ggez.mkv`, post-processing fails (`Missed file: /downloads/Foundation.S01E02.HDR.2160p.WEB.H265-GGEZ[rarbg]/foundation.s01e02.hdr.2160p.web.h265-ggez.mkv: Processing failed: This show isn't in your list, you need to add it before post-processing an episode`)

**To Reproduce**
1. Post-process a file named `foundation.s01e02.hdr.2160p.web.h265-ggez.mkv` for a series with the title `Foundation (2021)`

**Expected behavior**
The file `foundation.s01e02.hdr.2160p.web.h265-ggez.mkv` to be recognized and parsed for the series `Foundation (2021)`

**Screenshots**

**Medusa (please complete the following information):**
- Version: 0.5.17
 - OS: Linux-5.11.0-31-generic-x86_64-with
 - Branch: master
 - Commit: Unknown
 - Python version: 3.8.10 (default, May 6 2021, 06:30:44) [GCC 9.3.0]
 - Database version: 44.18

**Debug logs (at least 50 lines):**
General > Advanced Settings > Enable debug
```
Processing path: /downloads
No processable items found in folder: /downloads
No resource_name passed, using path [/downloads/Icarus.2017.1080p.WEBRip.x265-RARBG] to process as a source folder
Processing folder: /downloads/Icarus.2017.1080p.WEBRip.x265-RARBG/Subs
Post-processing files: ['2_English.srt', '3_English.srt', '4_Arabic.srt', '5_Chinese.srt', '6_French.srt', '7_Polish.srt']
Post-processing video files: []
Processing folder: /downloads/Icarus.2017.1080p.WEBRip.x265-RARBG
Post-processing files: ['Icarus.2017.1080p.WEBRip.x265-RARBG.mp4', 'RARBG.txt', 'RARBG_DO_NOT_MIRROR.exe']
Post-processing video files: ['Icarus.2017.1080p.WEBRip.x265-RARBG.mp4']
Found unwanted files: ['RARBG.txt', 'RARBG_DO_NOT_MIRROR.exe']
Processing failed for /downloads/Icarus.2017.1080p.WEBRip.x265-RARBG/Icarus.2017.1080p.WEBRip.x265-RARBG.mp4: This show isn't in your list, you need to add it before post-processing an episode
No resource_name passed, using path [/downloads/Foundation.S01E02.HDR.2160p.WEB.H265-GGEZ[rarbg]] to process as a source folder
Processing folder: /downloads/Foundation.S01E02.HDR.2160p.WEB.H265-GGEZ[rarbg]
Post-processing files: ['RARBG.txt', 'RARBG_DO_NOT_MIRROR.exe', 'foundation.s01e02.hdr.2160p.web.h265-ggez.mkv', 'foundation.s01e02.hdr.2160p.web.h265-ggez.nfo']
Post-processing video files: ['foundation.s01e02.hdr.2160p.web.h265-ggez.mkv']
Found unwanted files: ['RARBG.txt', 'RARBG_DO_NOT_MIRROR.exe', 'foundation.s01e02.hdr.2160p.web.h265-ggez.nfo']
Processing failed for /downloads/Foundation.S01E02.HDR.2160p.WEB.H265-GGEZ[rarbg]/foundation.s01e02.hdr.2160p.web.h265-ggez.mkv: This show isn't in your list, you need to add it before post-processing an episode
No resource_name passed, using path [/downloads/Foundation.S01E01.HDR.2160p.WEB.H265-GGEZ[rarbg]] to process as a source folder
Processing folder: /downloads/Foundation.S01E01.HDR.2160p.WEB.H265-GGEZ[rarbg]
Post-processing files: ['RARBG.txt', 'RARBG_DO_NOT_MIRROR.exe', 'foundation.s01e01.hdr.2160p.web.h265-ggez.mkv', 'foundation.s01e01.hdr.2160p.web.h265-ggez.nfo']
Post-processing video files: ['foundation.s01e01.hdr.2160p.web.h265-ggez.mkv']
Found unwanted files: ['RARBG.txt', 'RARBG_DO_NOT_MIRROR.exe', 'foundation.s01e01.hdr.2160p.web.h265-ggez.nfo']
Processing failed for /downloads/Foundation.S01E01.HDR.2160p.WEB.H265-GGEZ[rarbg]/foundation.s01e01.hdr.2160p.web.h265-ggez.mkv: This show isn't in your list, you need to add it before post-processing an episode
Problem(s) during processing, failed for the following files/folders: 
Missed file: /downloads/Icarus.2017.1080p.WEBRip.x265-RARBG/Icarus.2017.1080p.WEBRip.x265-RARBG.mp4: Processing failed: This show isn't in your list, you need to add it before post-processing an episode
Missed file: /downloads/Foundation.S01E02.HDR.2160p.WEB.H265-GGEZ[rarbg]/foundation.s01e02.hdr.2160p.web.h265-ggez.mkv: Processing failed: This show isn't in your list, you need to add it before post-processing an episode
Missed file: /downloads/Foundation.S01E01.HDR.2160p.WEB.H265-GGEZ[rarbg]/foundation.s01e01.hdr.2160p.web.h265-ggez.mkv: Processing failed: This show isn't in your list, you need to add it before post-processing an episode
```
</details>

**Additional context**
You guys created a rock ass program ❤️  Been using it for a long time to great effect. Thanks!!!

**Comment by p0psicles** [comment]
Add a scene exception for the title without year

**Comment by Zjaaspoer** [comment]
That works, thanks! I went through all the settings and also saw `Scene Exception` but didn't understand this was the way it should be used. In all honesty, I assumed it had something to do with scenes in the show (I didn't get much further than that).

I found a better explanation here: https://github.com/pymedusa/Medusa/wiki/Scene-exceptions-and-numbering

Maybe the naming and/or explanation in the interface can be improved a bit?

Anyway, thanks for the quick reply and the fix! 🙏
